src/platform/common/platform/fileSystem.node.ts

```diff
@@ -5,23 +5,36 @@ import * as tmp from 'tmp';
 import { promisify } from 'util';
 import * as vscode from 'vscode';
 import { traceError } from '../../logging';
-import { isFileNotFoundError } from './errors.node';
+import { isFileNotFoundError } from './errors';
 import { convertFileType, convertStat, getHashString } from './fileSystemUtils.node';
 import { TemporaryFile } from './types';
 import { IFileSystemNode } from './types.node';
 import { FileSystem as FileSystemBase } from './fileSystem';
+import { EXTENSION_ROOT_DIR } from '../../constants.node';
 
 /**
  * File system abstraction which wraps the VS Code API.
  */
 @injectable()
 export class FileSystem extends FileSystemBase implements IFileSystemNode {
     private globFiles: (pat: string, options?: { cwd: string; dot?: boolean }) => Promise<string[]>;
+
     constructor() {
         super();
+        this.rootDirectory = EXTENSION_ROOT_DIR;
```

Comment:
I don't think this is necessary. We should just use IExtensionContext as I mentioned before.